plugins/services/src/js/service-configuration/ServiceStorageConfigSection.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
```

Comment:
This is `ServiceConfigStorageSectionDisplay.js` and `StorageServiceConfigSection.js` combined